placetypes/building.json

```diff
@@ -1,9 +1,9 @@
 {
-	"wof:id": 102312327,
+	"wof:id": 1260214523,
 	"wof:name": "building",
 	"wof:role": "common_optional",
 	"wof:parent": [ "address", "intersection", "campus", "microhood", "neighbourhood" ],
 	"wof:concordances": {
-		"wd:id": "Q41176"
+		"wd:id": "Q116464161"
```

Comment:
This is not the WikiData ID for the concept of a building but for a locality in Guinea – https://www.wikidata.org/wiki/Q116464161